docs/api/browser-window.md

```diff
@@ -835,7 +849,7 @@ Moves window to `x` and `y`.
 
 #### `win.getPosition()`
 
-Returns an array that contains window's current position.
+Returns `Integer[]` - Contains the window's current position.
```

Comment:
Also, the parser shouldn't be modified to handle abstract modifiers to types.  The type should be fully contained, the less magic we have to add to the parser the better